krun/platform.py

```diff
@@ -21,21 +21,49 @@ def __init__(self):
         # errors and warnings in the Linux dmesg. If that happens, we
         # really want to know about it!
         self.current_dmesg = self._collect_dmesg_lines()
+        self.current_dmesg_time = time.time()
```

Comment:
I'm also not sure `current_dmesg_time` is a good name: shouldn't it be something like `last_dmesg_time`?